spotless/spotless.gradle

```diff
@@ -6,6 +6,11 @@ spotless {
         target "**/*.kt"
         targetExclude "**/build/**/*.kt"
         ktlint()
+                .editorConfigOverride([
+                        "ktlint_standard_property-naming"                  : "disabled",
+                        "ktlint_standard_backing-property-naming"          : "disabled",
+                        "ktlint_function_naming_ignore_when_annotated_with": "Composable",
+                ])
```

Comment:
This + the bump are the two changes I did manually. The rest comes from running `spotlessApply`